### PR TITLE
Added random caption generator

### PR DESCRIPTION
**Pull request by ranjeet25**
### You can generate random captions and you can copy them to your clipboard by just clicking on the copy button 🔥⭐ 
Screenshots :

image

Mobie View 

image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (epicdeepak) [inline .vscode/settings.json]: delete this file, it is not required.

**Review by epicdeepak — CHANGES_REQUESTED** [review]
Everything looks good, just delete a file and update the folder name.

File: .vscode/settings.json (hunk 1 above)
Comment (ranjeet25) [reply .vscode/settings.json]: @deeqakkk deleted !!

**Comment by ranjeet25** [comment]
> Everything looks good, just delete a file and update the folder name.

my project number is 6 so I have made folder according to it 

> 

My project number was 6 so I have named my folder according to it. @deeqakkk  you can verify !!

**Comment by ranjeet25** [comment]
@deeqakkk  Can you Please merge my Pull request !! I have made the changes

**Review by epicdeepak — APPROVED** [review]

**Comment by epicdeepak** [comment]
Good Work Ranjeet 👏🏼

**Pull request merged**